spec/components/Autocomplete.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react'
+
+interface AutocompleteProps {
```

Comment:
replace any types with correct types
(this is a test, I know, but still)
if we want to write typeless tests, there is always the js/jsx option